### DISCUSSION TITLE
Assign emojis for metadata items?

### DISCUSSION THREAD

**Discussion by Matthias84** [jonaswinkler/paperless-ng#565] [Feature Requests]
Hi, I like to suggest that users might assign emojis to documents contact, type and tags. Similar light coloured labels for tags, this might simplify to optical identification and save screen space when we show listings on limited devices or the commandline.

Slightly refers to #424  and #423

**Comment by jonaswinkler** [comment]
If we assign emojis for document types and correspondents, which emoji should we use when representing the document? Where should these icons be used, exactly?

**Comment by Matthias84** [comment]
I suggest to add an extra field to document tags, contacts, types models.

For my personal usecase, I would config it like:

* tags: work 🏢, privacy 🕴️, health 🥶, information requests 📰, OSM 🗺️, Freifunk 📡, registration ✍️, inbox 📥, insurance 🦺, living 🏘️)
* contacts: (logo / avatar support would be cool? otherwise genric banks  🏦, insurance 🦺, car 🚗, university 🎓, ...)
* types: forms ☑️, information ℹ️, recipe 🛒, summary 📋
![list emoji](https://user-images.githubusercontent.com/217511/105642708-9482d780-5e8b-11eb-8ffc-8106364ab068.png)
, contracts 

This symbols could be used instead of long colored labels, as they are easier to recognize?

**Comment by shamoon** [comment]
Its an interesting idea, so tags/dts/corrresps could all have an optional 'icon' field or something like that which can be either an emjoi, maybe an image.

It wouldn't be an insane amount of work but not trivial either, requires additional DB field presumably, images would need an uploading/storage mechanism so that'd be a bunch more work (could store image in db, set a small maximum size but that has pros / cons too).

The aesthetically-concerned side of me is a little troubled by the idea of a messy dashboard with all kinds of different emoji but Im not sure the bootstrap icons we're using already would have enough options.

**Comment by d8sychain** [comment]
> (could store image in db, set a small maximum size but that has pros / cons too)

I know little about this, but I think PostgreSQL can support images in the DB, however for SQLite users, a different approach would be needed.
